exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -198,20 +238,21 @@ impl ServiceFactory for JavaServiceFactory {
     }
 
     fn make_service(&mut self, context: &Context) -> Box<Service> {
-        let runtime = {
-            use exonum::helpers::fabric::keys;
-            let config: Config = context
-                .get(keys::NODE_CONFIG)
-                .expect("Unable to read node configuration.")
-                .services_configs
-                .get(super::cmd::EJB_CONFIG_SECTION_NAME)
-                .expect("Unable to read EJB configuration.")
-                .clone()
-                .try_into()
-                .expect("Invalid EJB configuration format.");
-            JavaServiceRuntime::get_or_create(config)
-        };
+        use exonum::helpers::fabric::keys;
+        let config: Config = context
+            .get(keys::NODE_CONFIG)
+            .expect("Unable to read node configuration.")
+            .services_configs
+            .get(super::cmd::EJB_CONFIG_SECTION_NAME)
+            .expect("Unable to read EJB configuration.")
+            .clone()
+            .try_into()
+            .expect("Invalid EJB configuration format.");
+        let runtime = { JavaServiceRuntime::get_or_create(config.clone()) };
```

Comment:
Can we instantiate runtime separately? It is weird that ServiceFactory that is supposed to have a simple dependency on JavaServiceRuntime resorts to such means of instantiating it 🍝 

If it is not possible now because of Exonum limitations — please document an issue for the core so that the runtime configuration works perfectly in dynamic services and assign to @aleksuss

Reply:
I've already mentioned such requirement several times on the wiki ☝️

Reply:
Mentioned on the wiki as discussed before.